backend/src/main/java/ca/gov/dtsstn/cdcp/api/data/repository/UserRepository.java

```diff
@@ -15,11 +15,7 @@ public interface UserRepository extends JpaRepository<UserEntity, String> {
 
 	List<UserEntity> findByEmail(String email);
 
-	@Query("""
-		SELECT user FROM User user JOIN FETCH user.userAttributes userAttribute
-		 WHERE userAttribute.name=:name
-		   AND userAttribute.value=:value
-	""")
+	@Query("SELECT user FROM User user JOIN FETCH user.userAttributes userAttribute WHERE userAttribute.name=:name AND userAttribute.value=:value")
```

Comment:
I prefer the whitespace for readability, and added whitespace causes no issues for JPA/Hibernate.

If we settle on a convention where all whitespace is removed from JPQL queries, if we add more complex queries in the future they're gonna be *extremely* difficult to read.

For an example of how complex these things can get, look at <https://github.com/DTS-STN/passport-status-api/blob/main/src/main/java/ca/gov/dtsstn/passport/api/data/PassportStatusRepository.java> (and these aren't even particularly complex queries).

Reply:
I agree with Grégoire

Reply:
Okay, I'll just close this PR then.